packages/samples/test/samples.test.ts

```diff
@@ -0,0 +1,34 @@
+import { resolvePath } from "@typespec/compiler";
+import { rm } from "fs/promises";
+import { fileURLToPath } from "url";
+import { defineSampleSnaphotTests } from "../src/sample-snapshot-testing.js";
+
+const excludedSamples = [
+  // fails compilation by design to demo language server
+  "local-typespec",
+
+  // no actual samples in these dirs
+  "node_modules",
```

Comment:
as there is already a lot of renames wondering if we should just not move the samples one folder deep so we don't need to exclude all those extra folders.